rotkehlchen/api/rest.py

```diff
@@ -4253,11 +4253,20 @@ def clear_avatars_cache(self, avatars: Optional[list[str]]) -> Response:
                 if entry.is_file():
                     entry.unlink()
 
+            with self.rotkehlchen.data.db.user_write() as delete_cursor:
+                delete_cursor.execute('UPDATE ens_mappings SET last_avatar_update=?', (Timestamp(0),))  # noqa: E501
```

Comment:
Isn't it better to set it to null? What was the thinking of having it as null before @nebolax ?

Reply:
I used null to be more explicit

Reply:
Alright then. Will set it to NULL